frontend/fingertips-frontend/components/views/OneIndicatorTwoOrMoreAreasView/index.tsx

```diff
@@ -3,29 +3,69 @@ import { areaCodeForEngland } from '@/lib/chartHelpers/constants';
 import { SearchParams, SearchStateManager } from '@/lib/searchStateManager';
 import { connection } from 'next/server';
 import { ViewProps } from '../ViewsContext';
+import { ApiClientFactory } from '@/lib/apiClient/apiClientFactory';
+import { HealthDataForArea } from '@/generated-sources/ft-api-client';
+import { SearchServiceFactory } from '@/lib/search/searchServiceFactory';
+import { IndicatorDocument } from '@/lib/search/searchTypes';
 
 export default async function OneIndicatorTwoOrMoreAreasView({
   searchState,
 }: Readonly<ViewProps>) {
   const stateManager = SearchStateManager.initialise(searchState);
   const {
     [SearchParams.AreasSelected]: areasSelected,
+    [SearchParams.IndicatorsSelected]: indicatorSelected,
     [SearchParams.GroupSelected]: selectedGroupCode,
   } = stateManager.getSearchState();
 
-  if (!areasSelected || areasSelected.length < 2) {
+  if (
+    indicatorSelected?.length !== 1 ||
+    !areasSelected ||
+    areasSelected?.length < 2
+  ) {
     throw new Error('Invalid parameters provided to view');
   }
 
-  const areaCodesToRequest = [...areasSelected, areaCodeForEngland];
+  const areaCodesToRequest = [...areasSelected];
+  if (!areaCodesToRequest.includes(areaCodeForEngland)) {
+    areaCodesToRequest.push(areaCodeForEngland);
+  }
   if (selectedGroupCode && selectedGroupCode != areaCodeForEngland) {
     areaCodesToRequest.push(selectedGroupCode);
   }
 
   await connection();
+  const indicatorApi = ApiClientFactory.getIndicatorsApiClient();
+
+  let healthIndicatorData: HealthDataForArea[] | undefined;
+  try {
+    healthIndicatorData = await indicatorApi.getHealthDataForAnIndicator({
+      indicatorId: Number(indicatorSelected),
```

Comment:
Isn't `indicatorSelected` an array here? Should this be:
```suggestion
      indicatorId: Number(indicatorSelected[0]),
```

Reply:
corresponding update also made to in OneIndicatorOneAreaView